packages/theme-chalk/src/dropdown.scss

```diff
@@ -137,9 +137,8 @@
     }
 
     @include when(disabled) {
-      cursor: default;
+      cursor: not-allowed;
       color: $--font-color-disabled-base;
-      pointer-events: none;
```

Comment:
You see, `pointer-evens: none;` actually prevents browser to dispatch mouse events on the node. You don't really need to add the javascript code above. Add this line back should be great.

Reply:
if not delete the line `pointer-evens: none;`,`cursor: not-allowed` will not work.so I delete it,using JS to replace it.How to fix this bug while retaining it